src/components/Tool/Appeal/Appeals.tsx

```diff
@@ -21,17 +26,44 @@ interface Props {
 
 const Appeals: React.FC<Props> = ({ accountListId }: Props) => {
   const { t } = useTranslation();
+  const { enqueueSnackbar } = useSnackbar();
+  const [
+    changePrimaryAppeal,
+    { loading: updating },
+  ] = useChangePrimaryAppealMutation();
   const { data, loading } = useGetAppealsQuery({
     variables: { accountListId },
   });
 
+  const changePrimary = async (newPrimaryId: string): Promise<void> => {
```

Comment:
```suggestion
  const changePrimary = async (newPrimaryId: string): void => {
```
If TS wants it to be a promise, that's fine. Just seems like an unnecessary detail since the return value  is never used.

Reply:
Yeah TS gives error:

The return type of an async function or method must be the global Promise<T> type. Did you mean to write 'Promise<void>'?ts(1064)

but changing it in the props to void is fine